build.sbt

```diff
@@ -17,7 +17,7 @@ lazy val core = (project in file ("modules/core"))
   .settings(
     name := "arugula-core",
     libraryDependencies ++= Seq(
-      "io.lettuce" % "lettuce-core" % "6.0.0+",
+      "io.lettuce" % "lettuce-core" % "6.3.0.RELEASE+",
```

Comment:
Do we still need the "+" as suffix here ? Not sure if it will work as intended... ?

Reply:
I think we probably need to experiment here. I notice on Maven, their official versions all end in `.RELEASE`

https://mvnrepository.com/artifact/io.lettuce/lettuce-core

and when I tried to omit it, I ran into an error.

Reply:
```
[IJ]reload
[info] welcome to sbt 1.10.1 (Amazon.com Inc. Java 21.0.4)
[info] loading global plugins from /Users/johnloehrer/.sbt/1.0/plugins
[info] loading settings for project arugula-build from idea1.sbt ...
[info] loading project definition from /Users/johnloehrer/code/arugula/project
[info] loading settings for project root from build.sbt ...
[info] set current project to arugula (in build file:/Users/johnloehrer/code/arugula/)
[IJ]testAll
[warn] 
[warn] 	Note: Unresolved dependencies path:
[error] stack trace is suppressed; run last core / update for the full output
[error] (core / update) sbt.librarymanagement.ResolveException: Error downloading io.lettuce:lettuce-core:6.3.0+
[error]   No version found for [6.3.0,6.3.0.max] in 
[error]   No version found for [6.3.0,6.3.0.max] in 
[error]   not found: /Users/johnloehrer/.ivy2/localio.lettuce/lettuce-core
[error]   No version found for [6.3.0,6.3.0.max] in https://repo1.maven.org/maven2/io/lettuce/lettuce-core/maven-metadata.xml
[error] Total time: 1 s, completed Jul 25, 2024, 4:30:16 PM
[IJ]
```

Reply:
using `6.3.+` seems to work. and more in line w what I think we want